NGitLab.Mock/ReleaseCollection.cs

```diff
@@ -40,13 +41,21 @@ public override void Add(ReleaseInfo release)
             base.Add(release);
         }
 
-        public ReleaseInfo Add(string tagName, string name, string description, User user)
+        public ReleaseInfo Add(string tagName, string name, string reference, string description, User user)
```

Comment:
This is a breaking change. Should we add a method overload instead?

@meziantou , what do you think?

Reply:
(on mocks, I'm not sure we care)

Reply:
(would be nice to add https://www.nuget.org/packages/Microsoft.CodeAnalysis.PublicApiAnalyzers)

Reply:
I've reintroduced the previous method, so there is no breaking change.

@jairbubbles https://github.com/ubisoft/NGitLab/issues/52